requirements-dev.txt

```
@@ -1,4 +1,4 @@
-paho-mqtt>=1.5.0,<2.0.0
+paho-mqtt>=2.0.0,<3.0.0
```

```suggestion
paho-mqtt>=2.1.0,<3.0.0
```

This will fix the compatibility issues with HA Core 2025.3